nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/resources/OSGI-INF/comment-notification-contrib.xml

```diff
@@ -40,4 +40,9 @@
     <veto name="CommentNotificationVeto" class="org.nuxeo.ecm.platform.comment.notification.CommentNotificationVeto" />
   </extension>
 
+  <extension target="org.nuxeo.runtime.ConfigurationService"
+    point="configuration">
+    <property name="org.nuxeo.ecm.platform.comment.service.notification.autosubscribe">false</property>
```

Comment:
Should be `true` by default on `master`, according to @aescaffre comment https://jira.nuxeo.com/browse/NXP-28254?focusedCommentId=421233&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-421233

Reply:
Added the test notification-contrib.xml to the CommentFeature so all tests using it can disable autosubscribe.
The test subscription to re enable subscriptions is still in use.